data/presets/presets/shop/general.json

```diff
@@ -0,0 +1,12 @@
+{
+    "icon": "maki-shop",
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "tags": {
+        "shop": "general"
+    },
+    "terms": ["village shop"],
+    "name": "General store"
```

Comment:
capitalize like "General Store"

Reply:
Thanks.. The terms don't have to be capitalized, they are for searching, not display.